src/scroll-timeline-css-parser.js

```diff
@@ -22,13 +25,16 @@ const ANIMATION_KEYWORDS = [
   'ease', 'linear', 'ease-in', 'ease-out', 'ease-in-out'
 ];
 
+const VIEW_TIMELINE_AXES = ['block', 'inline',  'vertical', 'horizontal'];
```

Comment:
VIEW_TIMELINE_AXIS_(NAMES|TYPES)?

VIEW_TIMELINE_AXES and VIEW_TIMELINE_AXIS are close enough that it can make the code harder to read.